test/uk/gov/hmrc/nisp/views/TermsAndConditionsViewSpec.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.nisp.views
+
+import org.scalatest.mockito.MockitoSugar
+import play.api.test.FakeRequest
+import play.api.test.Helpers._
+import uk.gov.hmrc.nisp.controllers.auth.NispAuthedUser
+import uk.gov.hmrc.nisp.fixtures.NispAuthedUserFixture
+import uk.gov.hmrc.nisp.helpers._
+import uk.gov.hmrc.nisp.utils.Constants
+import uk.gov.hmrc.play.partials.FormPartialRetriever
+import uk.gov.hmrc.renderer.TemplateRenderer
+
+class TermsAndConditionsViewSpec extends HtmlSpec with MockitoSugar {
+
+  val fakeRequest = FakeRequest("GET", "/")
+
+  implicit val cachedStaticHtmlPartialRetriever = FakeCachedStaticHtmlPartialRetriever
+  implicit val formPartialRetriever: FormPartialRetriever = FakePartialRetriever
+  implicit val templateRenderer: TemplateRenderer = FakeTemplateRenderer
+  implicit val user: NispAuthedUser = NispAuthedUserFixture.user(TestAccountBuilder.regularNino)
+
+  val feedbackFrontendUrl: String = "/foo"
+  lazy val html = uk.gov.hmrc.nisp.views.html.termsAndConditions(true)
+  lazy val source = asDocument(contentAsString(html))
+
+  "TermsAndConditions" should {
+
+    "assert correct page title" in {
+      val title = source.title()
+      val expected = messages("nisp.tandcs.title") + Constants.titleSplitter +
+        messages("nisp.title.extension") + Constants.titleSplitter + messages("nisp.gov-uk")
+      title must include(expected)
+    }
+
+    "assert correct heading title on page" in {
+      val title = source.getElementsByTag("h1").get(0).toString
```

Comment:
This is being a little bit pedantic, you could use different selectors. Ideally I prefer using ID's when possible or classes

Reply:
Thanks John, I think most of this will be carried out within the refactor one the views are then injected.